tools/sense_studio/sense_studio.py

```diff
@@ -268,5 +268,14 @@ def add_header(r):
     return r
 
 
+@app.context_processor
+def class_labels_processor():
+    def inject_class_labels(project):
+        path = utils.lookup_project_path(project)
+        labels = utils.get_class_labels(path)
+        return labels
+    return dict(inject_class_labels=inject_class_labels)
```

Comment:
Very interesting :thinking: Reading the example on https://flask.palletsprojects.com/en/1.1.x/templating/#context-processors, we might also have used some global scope to store the current `project` name, but your solution also works. Maybe just add a comment on how this works and is used, so others can understand more easily.

Reply:
sure